src/components/elements/dropdown.js

```diff
@@ -1,11 +1,243 @@
-import styled from 'styled-components'
+import React, { useRef, useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+import Chevron from 'images/svg/chevron-bottom.svg'
 
-const StyledSelect = styled.select`
+const DropdownContainer = styled.ul`
+    list-style: none;
+    position: relative;
+    border: 1px solid var(--color-grey-7);
+    cursor: pointer;
+    padding: 0;
+    border-radius: 4px;
+    width: 15.2rem;
+    height: 3.2rem;
+    top: 25%;
+    ${props => props.active && 'border-color: var(--color-green) !important;'}
+
+    &:hover {
+        border-color: var(--color-grey-5);
+    }
+`
+
+const DropdownSelected = styled.li`
     color: var(--color-black-3);
-    border: 1px solid var(--color-grey-1);
-    padding: 1rem;
-    margin: 1rem;
-    background-color: 1px solid var(--color-grey-1);
+    list-style-position: inside;
+    white-space: nowrap;
+    overflow: hidden;
+    padding: 0 1.6rem;
+    text-overflow: ellipsis;
+    height: 100%;
+    font-size: var(--text-size-xs);
+    display: flex;
+    align-items: center;
+
+    &:focus {
+        outline: none;
```

Comment:
this is as per the styleguide?

Reply:
by default, there will be blue2 outline if got focus. maybe can make it the same as active which is green border

Reply:
would be nice to have it in the official styleguide https://a11yproject.com/posts/never-remove-css-outlines/